src/Controls/src/Core/Page/Page.cs

```diff
@@ -56,6 +56,7 @@ public partial class Page : VisualElement, ILayout, IPageController, IElementCon
 		public static readonly BindableProperty BackgroundImageSourceProperty = BindableProperty.Create(nameof(BackgroundImageSource), typeof(ImageSource), typeof(Page), default(ImageSource));
 
 		/// <summary>Bindable property for <see cref="IsBusy"/>.</summary>
+		[Obsolete("Page.IsBusy has been deprecated and will be removed in .NET 11")]
```

Comment:
@davidortinau do we need to put links to docs in here? Maybe an aka.ms/maui/deprecations/isbusy and then we say why in a gh issue or something? Or link to release notes. The aka can be changed once the notes are created.